index.ts

```diff
@@ -1,30 +1,40 @@
-// @flow
-
 import GeocoderArcGIS from 'geocoder-arcgis'
-import lonlat from '@conveyal/lonlat'
+import type { GeoJSON } from 'geojson'
+import { toCoordinates, toString, toPoint } from '@conveyal/lonlat'
+import type { LonLatInput } from '@conveyal/lonlat'
+
+type Point = {
+  x: number
+  y: number
+}
 
-const arcGisGeocoders = {}
+type GeocodersMap = {
+  [key: string]: GeocoderArcGIS
+}
 
 type Boundary = {
   rect: {
-    minLat: number,
-    minLon: number,
-    maxLat: number,
+    minLat: number
+    minLon: number
+    maxLat: number
     maxLon: number
   }
 }
 
+type Candidate = {
+  location: LonLatInput
+  attributes: Record<string, string | number>
+}
```

Comment:
It's probably fine to not spell out every exact attribute here, but it would be nice to include some additional comments. Also, the location key while parseable by lonlat, has a more specific type. Recommended change:

```suggestion
/**
 * A geocoding candidate as received from either the findAddressCandidates or geocodeAddresses endpoint.
 * See info about some attributes here: https://developers.arcgis.com/rest/geocode/api-reference/geocoding-service-output.htm
 */
type Candidate = {
  location: {
    x: number
    y: number
  }
  attributes: Record<string, string | number>
}
```

Reply:
Thanks for the suggestion! Added in aecf758d679984328a465df451fd953bde17efbf